api/specs/director/openapi.yaml

```diff
@@ -128,6 +128,41 @@ paths:
               schema:
                 $ref: "#/components/schemas/ErrorEnveloped"
 
+  /services/{service_key}/{service_version}:labels:
```

Comment:
labels is not a custom method but a sub-resource, therefore it should be ``/services/{service_key}/{service_version}/labels`` with ``/`` not ``:``

I thought we agreed on this, **please** check [guidelines](https://cloud.google.com/apis/design/custom_methods)

Reply:
replaced.